garak/resources/autodan/genetic.py

```diff
@@ -278,13 +279,13 @@ def gpt_mutate(mutation_generator, sentence: str) -> str:
     while not received:
         try:
             # TODO: Make the model configurable.
-            response = mutation_generator.generate(
-                prompt=[
-                    {"role": "system", "content": system_msg},
-                    {"role": "user", "content": user_message},
-                ]
-            )
-            revised_sentence = response[0].replace("\n", "")
+            conv = Conversation(turns=[
+                Turn(role="system", content=Message(text=system_msg)),
+                Turn(role="user", content=Message(text=user_message)),
+            ])
+            response = mutation_generator.generate(prompt=conv)
+            response_text = response[0].text if isinstance(response[0], Message) else str(response[0])
+            revised_sentence = response_text.replace("\n", "")
             received = True
```

Comment:
Similar to other comment:
```suggestion
            response = mutation_generator.generate(prompt=conv)[0]
            if response and response.text:
                revised_sentence = response.text.replace("\n", "")
                received = True
```